bin/git-reauthor

```diff
@@ -22,7 +23,8 @@ Options
     -c, --use-config              define correct values from user Git config
     -e, --correct-email  <email>  define the correct email to set
     -n, --correct-name   <name>   define the correct name to set
-    -o, --old-email      <email>  rewrite identities matching old email in commits and tags
+    -oe, --old-email     <email>  rewrite identities matching old email in commits and tags
```

Comment:
It would be great to keep the old option. For example, `-oe, --old-email, -o`.
And don't forget to update the man page and Commands.md, so that everyone can find your new feature!